### PR TITLE
add aggregators section to merchants page.

### LINKED ISSUE #1970: Listing Swapzone.io on "Swappers" section
Hello team!

I'm Maria, a marketing manager at Swapzone - an instant crypto exchange aggregator. We help users to compare exchange rates, analyze and swap cryptocurrency in just one interface. 

XMR is one the most popular asset on our platform and we also have a separate page of your coin so new users can learn about your project: https://swapzone.io/currencies/monero

I'd love to know whether it's possible for you to add our project in your "Swappers" section (https://www.getmonero.org/community/merchants/). It would be a huge deal for us. 

Let me know your opinion about it!
Best regards, Maria

### PR DESCRIPTION
**Pull request by HardenedSteel**
Added aggregators section to merchants page and added two providers for aggregators section; Orangefren and Swapzone.io
closes #1970

### CODE REVIEW

File: _i18n/en.yml [hunk 1]
```diff
@@ -153,6 +153,7 @@ merchants:
   centrexchangesp: If you prefer to use centralized exchanges, here is a list of renowned CEXes and swappers. Many more exchanges support Monero, we list here only a few reputable ones.
   cexp: Centralized exchanges that offer exchanging Monero for national currencies and cryptocurrencies.
   swappersp: Swappers let users exchange XMR for other cryptocurrencies.
+  aggregatorsp: Aggregators compare all exchanges at once and find cheapest, fastest or best rated exchanges.
```
Comment (erciccione) [inline _i18n/en.yml:156]: Is it all "exchanges" or all "swappers"?

**Comment by plowsof** [comment]
Note: there is now trocador.app now , whose main advantage over orengefren.com is that it does not use javascript - and the UX is better as swaps happen on the same website.

File: _i18n/en.yml (hunk 1 above)
Comment (HardenedSteel) [reply _i18n/en.yml:156]: swappers*

**Comment by ssvdrevg** [comment]
Hi there, any updates on merging this PR?
There are more and more Monero fans on Swapzone:)

**Review by ChanarinThintip — COMMENTED** [review]
Good

**Comment by MajesticBank** [comment]
I think both 3 are very interesting services but it's disrespectful listing short running Aggregators on top of Swappers (exchanges) which supported this community for more then 5 years.

Exchanges also have liability for customers privacy and provide actual service, invest large amount of funds in hosting and securing user data. Aggregators where swaps happen on site as "proxy" should be less trusted because they can actually observe customers deposits and in general it's lower privacy solution. 

Also in my opinion any service should be running for at least 1 year before being considered listed on getmonero.org website. Since Aggregators  can easily add new exchange to their website minimum should be up to 2 years to avoid getmonero.org being involved with some kind of scam or rug pull. 

swapzone.io - 2 years (not sure how much relevant for monero community(
orangefren.com - 1 year (barely on the border for consideration) https://www.reddit.com/r/Monero/comments/s5hrgw/orangefrencom_instant_exchange_comparison_with/
trocador.app - 4 months ( should be considered in at least 8 months from now) https://www.reddit.com/r/Monero/comments/y235u6/cryptomorpheus_is_launching_an_instant_exchange/


If I can agree on anything that's listing kycnot.me, as it's most unbiased aggregators listing least/not based on monetizing model but good old community thing, has long track running record ( https://www.reddit.com/r/Monero/comments/pgcdae/kycnotme_a_list_of_nonkyc_crypto_services/ ) and show real privacy nature of swap services/exchanges but however below still Swappers (exchanges).


Regardless is it majesticbank, stackwallet or rino, there should be much longer waiting and vetting period before service is listed on canonical monero website.

**Comment by erciccione** [comment]
> Also in my opinion any service should be running for at least 1 year before being considered listed on getmonero.org website

I agree.

**Comment by ssvdrevg** [comment]
>...in general it's lower privacy solution.

All the swaps on SZ are custody-free, with no registration needed. Anonymity and privacy of each transaction are the integral components of the project concept. 

> swapzone.io - 2 years (not sure how much relevant for monero community(

For 2 years, Swapzone has been collecting the offers for $XMR swaps from vary reliable exchangers, each of which is thoroughly tested and reviewed. The stable work of partners has been proven by time - all providers have been working with us for **at least a year**. And all this time, users have been making informed choices with Swapzone. 

The increasing popularity of Monero on Swapzone only confirms the relevance of the listing for the community, so I kindly ask you to reconsider the idea of mentioning Swapzone in this section.

Thank you!

**Review by nahuhh — CHANGES_REQUESTED** [review]
Can you add tor and i2p addresses for those who support
example 

orangefren:
http://h74rtjzibohtgaehqyvy3cquqrsgn3pyxxqzwiw3u7mxal6n66ra.b32.i2p/
http://rnwis2whetqcj4oknksnc5l24jbh33nflunifff3xtjjonnoxu3ld6id.onion/

trocador:
trocador.i2p
http://trocadorfyhlu27aefre5u7zri66gudtzdyelymftvr4yjwcxhfaqsid.onion/

**Review by nahuhh — APPROVED** [review]
lgtm. Just merge conflicts

**Comment by MajesticBank** [comment]
My comment from year and half ago aged really well on this. 
"Since Aggregators can easily add new exchange to their website minimum should be up to 2 years to avoid getmonero.org being involved with some kind of scam or rug pull."

Loss of funds is not something I can support, there must be liability for w/e sites you promote / work with. 
Either aggregators must provide full insurance or limit transactions per 2k or how much they can give user in case of scam.
Selective scamming is not OKAY under KYC shield like changelly and freewallet is doing, this is no different story.

As an example: https://www.reddit.com/r/Monero/comments/1d8olsd/swapter_225_xmr_missing/?rdt=47940&sort=new

Quick search on "swapter scam" 

https://www.reddit.com/r/monerosupport/comments/1bk3qde/warning_swapspaceswapter/ 200 usd
https://www.reddit.com/r/Monero/comments/17w3i71/swapterio_funds_scammed_beware/ 50 eth - 160 k USD

https://www.trustpilot.com/review/swapter.io?stars=1 >10% bad feedback

Swapter  is still listed on Trocador and Orangefren but seems to be removed from Swapzone ( So we have to wait that Swapter  pull 100 selective scams to be de-listed from all the aggregators ? )

Until recently there was warning on irc and still on reddit about Changelly which is listed https://swapzone.io/,  it doesn't make sense providing traffic to Changelly.

The list of same looks sites from same script and same feedback is not so small. 

https://www.trustpilot.com/review/exolix.com?stars=1 >10% bad feedback

https://www.reddit.com/r/Monero/comments/11hgrl5/warning_exolixcom_cryptocurrency_exchange_kept_my/?sort=confidence

https://www.reddit.com/r/Monero/comments/j428i2/warning_exolixcom_has_been_advertising_here_and/g7i3yu2/

Listed on Trocador and Orangefren 

https://www.trustpilot.com/review/swapuz.com?stars=1

Listed on Trocador

Who want to spend a bit more time can dig a lot more stuff.

TLDR Bringing traffic and legitimacy to selective-scam sites on narrative of just "facilitating" the swap which obviously have history on "Loss of funds" is not something I would like to see on getmonero website.

I am not against, just the aggregators need to clean the house and figure business model with liability where user can't lose it's funds.

Points from my previous comment are still actual.

**Comment by HardenedSteel** [comment]
> "Please note: these links are being provided as a convenience and for informational purposes only; they do not constitute an endorsement by the Monero community of any products, services or opinions of the corporations or organizations or individuals listed. The Monero community bears no responsibility for the accuracy, legality, or content of these external sites. Contact the external site for answers to questions regarding its content. As always, caveat emptor ('buyer beware'); you are responsible for doing your own research. Always use judgement when making online purchases." 

> Swapter is still listed on Trocador and Orangefren but seems to be removed from Swapzone

can't verify
